examples/services/fabricv4/cloud_router/cloud_router.py

```diff
@@ -0,0 +1,93 @@
+from examples.services.fabricv4.utils import utils
+from equinix.services import fabricv4 as module
```

Comment:
Can we import just as fabricv4 and not as module? Better for code readability to have a specific name other than an abstract one.